app-dicts/myspell-en/myspell-en-20181101.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_PV="${PV:0:4}-${PV:4:2}.${PV:6:2}"
+
+inherit myspell-r2
+
+DESCRIPTION="English dictionaries for myspell/hunspell"
+HOMEPAGE="https://extensions.libreoffice.org/extensions/english-dictionaries"
+SRC_URI="https://extensions.libreoffice.org/extensions/english-dictionaries/${MY_PV}/@@download/file/dict-en-${PV}.oxt"
+
+LICENSE="BSD MIT LGPL-3"
+SLOT="0"
+KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 ~mips ppc ppc64 sh sparc x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
```

Comment:
Why the stable keywords?

Reply:
same as previous versions, why change them?

Reply:
https://devmanual.gentoo.org/keywording/index.html

Reply:
so are you proposing I mark the 3 year old version as stable and leave this new version keyworded?

Reply:
i added back current stable release from drop commit. and changed the keywording here